### PR TITLE
Add DXVK Async patches

### PR DESCRIPTION
**Pull request by steamp0rt**
incase anyone doesn't like stuttering in OW/PoE

### CODE REVIEW

File: wine-tkg-git/customization.cfg [hunk 1]
```diff
@@ -92,7 +92,7 @@ _use_pba="true"
 _use_vkd3d="false"
 
 # dxvk - Replaces wined3d d3d10 and d3d11 with winelib DXVK for d3d10 and d3d11 translation
-_use_dxvk="false"
+_use_dxvk="true"
```
Comment (Tk-Glitch) [inline wine-tkg-git/customization.cfg]: DXVK should never be enabled by default, especially in this winelib state

File: wine-tkg-git/PKGBUILD [hunk 2]
```diff
@@ -148,12 +148,22 @@ else
   fi
 
   if [ "$_use_dxvk" == "true" ] && [ "$_use_dxup" == "true" ]; then
-    pkgname="${pkgname/%-git/-dxvk+up-git}"
+		if [ "$_dxvk_async" == "true"]; then
```
Comment (Tk-Glitch) [inline wine-tkg-git/PKGBUILD]: That whole block seems unnecessary to me. Adding a tag instead would be enough.

File: wine-tkg-git/PKGBUILD [hunk 3]
```diff
@@ -1165,6 +1179,16 @@ package() {
 	  msg2 '##########################################################################################################################'
 	fi
 
+	if [ "$_dxvk_async" == "true" ]; then
```
Comment (Tk-Glitch) [inline wine-tkg-git/PKGBUILD]: You'd want to also check that `$_use_dxvk` value is `true`

File: wine-tkg-git/customization.cfg [hunk 4]
```diff
@@ -193,6 +193,9 @@ _stg_shared_mem_default="true"
 # Update winevulkan to whatever version I have pushed last, til next time, for new shiny and tasty vk extensions support. Thanks dadドイツ人 !
 _update_winevulkan="true"
 
+# Use the DXVK "Async" patch. WARNING: may cause anti-cheat systems to go off.
```
Comment (Tk-Glitch) [inline wine-tkg-git/customization.cfg]: Moving this next to other dxvk related options would make sense

File: wine-tkg-git/PKGBUILD (hunk 2 above)
Comment (Tk-Glitch) [reply wine-tkg-git/PKGBUILD]: I see now that you did add the tag. I'm convinced you should still consider dropping this block.

File: wine-tkg-git/customization.cfg (hunk 1 above)
Comment (steamp0rt) [reply wine-tkg-git/customization.cfg]: Whoops, my mistake

File: wine-tkg-git/PKGBUILD (hunk 2 above)
Comment (steamp0rt) [reply wine-tkg-git/PKGBUILD]: Dropped.

File: wine-tkg-git/PKGBUILD (hunk 3 above)
Comment (steamp0rt) [reply wine-tkg-git/PKGBUILD]: Done.

File: wine-tkg-git/customization.cfg (hunk 4 above)
Comment (steamp0rt) [reply wine-tkg-git/customization.cfg]: Done.

**Comment by Tk-Glitch** [comment]
All good to me :frog:

**Comment by steamp0rt** [comment]
Thanks!

**Comment by steamp0rt** [comment]
oh uh i made a woopies

```if [ "$_use_dxvk" == "true" && "$_dxvk_async" == "true" ]; then```
should be
```if [ "$_use_dxvk" == "true" ] && [ "$_dxvk_async" == "true" ]; then```

**Comment by Tk-Glitch** [comment]
No worries, it's fixed already ;)

**Comment by steamp0rt** [comment]
oh phew

**Comment by steamp0rt** [comment]
lol i can't test this with OW cuz it seems Wine 4.4 broke it. tested with both async and a 4.4 version from lutris

**Comment by Tk-Glitch** [comment]
Weird about the breakage as I don't remember seeing reports, but you can switch to any previous wine version from the .cfg if needed :)

**Comment by steamp0rt** [comment]
alright no it's my pc i think. or an OW update..? even 3.18 has the problem

**Comment by ronoverdrive** [comment]
The last time I tried building DXVK with the ASYNC patches it stopped working after version 0.96. Has this been fixed and available on 1.xxx?

**Comment by steamp0rt** [comment]
Problem was mesa-git. Was working peachy-clean til today's update. Guess that's the price of unstable builds.

@ronoverdrive the patch applies but i can't tell if it works

**Comment by Tk-Glitch** [comment]
Sh*t can happen at anytime with mesa git and LLVM :frog:

**Comment by steamp0rt** [comment]
yup. worked amazingly fine up until today.

**Pull request merged**